solutions/fully-configurable/main.tf

```diff
@@ -194,6 +195,22 @@ data "ibm_resource_instance" "existing_sm" {
   identifier = var.existing_secrets_manager_crn
 }
 
+module "secrets_manager_group" {
+  depends_on           = [module.secrets_manager]
```

Comment:
why the explicit depends_on? Its a bad practise, there is already an implicit dependency by passing the guid to this module